.JuliaFormatter.toml

```diff
@@ -0,0 +1,2 @@
+# See https://domluna.github.io/JuliaFormatter.jl/stable/ for a list of options
+style = "sciml"
```

Comment:
make sure to actually run the formatter before you merge the branch!

You tend to open a terminal, create temporary session `]activate --temp`; add and import JuliaFormatter and run `format(".")` in your project folder! that will format all files

(VSCode does it automatically for me on every file save)

Reply:
VSCode does it for me as well. I thought it was the Prettier extension. After disabling, it would still automatically format on every save